tests/e2e/extension/00-install.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: opentelemetry.io/v1beta1
+kind: OpenTelemetryCollector
+metadata:
+  name: jaeger-inmemory
+spec:
+  image: jaegertracing/jaeger-snapshot:latest
+  ports:
+  - name: jaeger
+    port: 16686
```

Comment:
Ideally, this should happen automatically.

Reply:
Removed it.